package.json

```diff
@@ -75,7 +75,8 @@
     "@material/top-app-bar": "0.39.3",
     "@material/typography": "0.39.0",
     "bind-decorator": "^1.0.11",
-    "material-components-web": "0.39.3"
+    "material-components-web": "0.39.3",
+    "serve": "^7.0.0"
```

Comment:
Does this have to be in the root `package.json`?

If not consider moving it to `docs/package.json`

Reply:
great point